### PR TITLE
[4.0] Fix notice in layouts/joomla/form/field/combo.php

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue #23119.

### Summary of Changes

Fixes notice when using combo field.

### Testing Instructions

Code review or add `combo` field to form. Open a page containing the form.
### Expected result

No notice.

### Actual result

```
Notice: Undefined variable: attr in /layouts/joomla/form/field/combo.php on line 47
```

### Documentation Changes Required
No.

### CODE REVIEW

File: layouts/joomla/form/field/combo.php [hunk 1]
```diff
@@ -44,6 +44,9 @@
 
 HTMLHelper::_('behavior.combobox');
 
+$attr = '';
```
Comment (Quy) [inline layouts/joomla/form/field/combo.php:47]: This will work or change line 50 to be the initial assignment.

Comment (SharkyKZ) [reply layouts/joomla/form/field/combo.php:47]: I'll leave it as it is, to be consistent with some other files:
```
layouts\joomla\form\field\accesslevel-fancy-select.php
layouts\joomla\form\field\accesslevel.php
layouts\joomla\form\field\list-fancy-select.php
layouts\joomla\form\field\list.php
layouts\joomla\form\field\media.php
layouts\joomla\form\field\tag.php
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b77c14411ebdb529d0453ece350854190c50b9ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23120">issues.joomla.org/tracker/joomla-cms/23120</a>.</sub>

**Comment by Quy** [comment]
2nd successful test as stated here https://github.com/joomla/joomla-cms/issues/23119#issuecomment-439949503<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23120">issues.joomla.org/tracker/joomla-cms/23120</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23120">issues.joomla.org/tracker/joomla-cms/23120</a>.</sub>

**Pull request merged**